### PR TITLE
v8 added SDF visual test

### PR DESCRIPTION
**Pull request by GoodBoyDigital**
added MSDF visual test
slight tweak for how fragments are organised

### CODE REVIEW

File: tests/visual/scenes/text/msdf-text.scene.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Assets } from '../../../../src/assets/Assets';
+import { Text } from '../../../../src/scene/text/Text';
+
+import type { Container } from '../../../../src/scene/container/Container';
+import type { TestScene } from '../../types';
+
+export const scene: TestScene = {
+    it: 'should render msdf-text correctly',
+    only: true,
```
Comment (Zyie) [inline tests/visual/scenes/text/msdf-text.scene.ts:9]:
```suggestion
```

File: tests/visual/scenes/text/sdf-text.scene.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Assets } from '../../../../src/assets/Assets';
+import { Text } from '../../../../src/scene/text/Text';
+
+import type { Container } from '../../../../src/scene/container/Container';
+import type { TestScene } from '../../types';
+
+export const scene: TestScene = {
+    it: 'should render sdf-text correctly',
+    only: true,
```
Comment (Zyie) [inline tests/visual/scenes/text/sdf-text.scene.ts:9]:
```suggestion
```

File: tests/visual/snapshots/webgl-msdf-text-scene-ts.png
Comment (Zyie) [inline tests/visual/snapshots/webgl-msdf-text-scene-ts.png:1]: These webgl tests are looking different from the webgpu tests

There is a black outline around them

Comment (GoodBoyDigital) [reply tests/visual/snapshots/webgl-msdf-text-scene-ts.png:1]: should be fixed now! thanks to #9806

**Review by Zyie — APPROVED** [review]

**Pull request merged**